src/modules/returns/lib/xml-to-json-mapping.js

```diff
@@ -41,19 +55,27 @@ const wasMeterUsed = (meterUsage) => {
 };
 
 const getOverallReadingType = (ret) => {
-  const meterUsage = ret.get('tns:GorPart', options).get('tns:FullReturnStructure', options).get('tns:MeterUsage', options);
-  if (wasMeterUsed(meterUsage)) return 'measured';
-  return 'estimated';
+  const meterUsage = ret
+    .get('tns:GorPart', options)
+    .get('tns:FullReturnStructure', options)
+    .get('tns:MeterUsage', options);
+
+  return wasMeterUsed(meterUsage) ? 'measured' : 'estimated';
 };
 
 const getUnits = (ret) => {
-  const unitOfMeasurement = ret.get('tns:GorPart', options).get('tns:FullReturnStructure', options).get('tns:UnitOfMeasurement', options).text();
-  if (unitOfMeasurement === 'CubicMetres') return 'm³';
-  return null;
+  const unitOfMeasurement = ret
+    .get('tns:GorPart', options)
+    .get('tns:FullReturnStructure', options)
+    .get('tns:UnitOfMeasurement', options)
+    .text();
+
+  return (unitOfMeasurement === 'CubicMetres') ? 'm³' : null;
```

Comment:
Are we certain that only CubicMetres is supported in the XML, or do we need to support other units